### PR TITLE
more_useful_clean_code

### PR DESCRIPTION
**Pull request by AvniErtas**
Yesterday I saw your package in Linkedln and I decided to use in my project. But when I try to use I found some issues for users. I decided to upgrade with using more clean code and sheet name change option. Also we can add later sheet count option (can be changed by user)

Best Regards,
Avni

### CODE REVIEW

File: lib/src/navigationBar.dart [hunk 1]
```diff
@@ -42,82 +65,31 @@ class _AwesomeNavigationBarState extends State<AwesomeNavigationBar> {
...
+  BottomNavigationBarItem buildBottomNavBarItem(
+      {required String icon, required String label, required String gifPath}) {
+    return BottomNavigationBarItem(
+      icon: Image.asset(
+        icon,
+        height: iconSize,
+        width: iconSize,
+      ),
+      label: label,
+      activeIcon: Image.asset(
```
Comment (Drishtantranjan) [inline lib/src/navigationBar.dart:88]: assets will not get loaded by using this way

File: lib/src/navigationBar.dart [hunk 2]
```diff
@@ -1,31 +1,54 @@
...
 
   @override
   _AwesomeNavigationBarState createState() => _AwesomeNavigationBarState();
 }
 
 class _AwesomeNavigationBarState extends State<AwesomeNavigationBar> {
+  late double iconSize;
+
+  final List<String> mainIconPaths = [
+    "assets/home/ic_home.png",
```
Comment (Drishtantranjan) [inline lib/src/navigationBar.dart:23]: home.png does not exist as asset

**Review by Drishtantranjan — CHANGES_REQUESTED** [review]
Please look into the reviews I have mentioned. Make sure to do the changes accordingly and submit the PR again.

**Comment by AvniErtas** [comment]
Hi,

Sorry for late answer but right now here is Sunday and honestly I have to spend time with my family so I don't have time to fix it right now but as soon as possible I will check it.

Best Regards,
Avni

Android için Outlook<https://aka.ms/AAb9ysg> edinin
________________________________

**Pull request merged**